rbd/watchers_mimic.go

```diff
@@ -0,0 +1,98 @@
+// +build !luminous,!mimic
```

Comment:
I think you want only `!luminous` here.

Reply:
Hmm, I must have pushed something before making my last changes... This is already gone in my local branch :-/